### PR TITLE
Add r-phylolm

### PR DESCRIPTION
**Pull request by apcamargo**
This PR adds the [`r-phylolm`](https://cran.r-project.org/web/packages/phylolm/index.html) R library.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by apcamargo** [comment]
@conda-forge/help-r What is the recommendation here? I couldn't find another recipe that depends on a minimum `r-base` version.

**Comment by jdblischak** [comment]
I think the current practice is to [skip the build](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build) when a recipe can't be built with one of the versions of Python or R pinned in the global [`conda_build_config.yaml`](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).

The R package that I can think off the top of my head that requires R >= 4.0 is [XML](https://cran.r-project.org/package=XML
), and the [PR](https://github.com/conda-forge/r-xml-feedstock/pull/17) to update [r-xml-feedstock](https://github.com/conda-forge/r-xml-feedstock) is using this strategy.

I recommend trying the following:

```
build:
  merge_build_host: True  # [win]
  skip: True  # [r_base < "4.0"]
  number: 0
```

If that doesn't work, then you can more explicitly exclude the R 3.6 build:

```
build:
  merge_build_host: True  # [win]
  skip: True  # [r_base != "3.6"]
  number: 0
```

Here's a search that shows existing recipes that skip a build based on the version of `r_base`: https://github.com/search?p=4&q=org%3Aconda-forge+r_base+skip&type=Code

**Comment by apcamargo** [comment]
Thanks, @jdblischak! It worked with `skip: True  # [r_base != "3.6.3"]`.

**Comment by jdblischak** [comment]
@apcamargo I'm glad that worked!

Other conda-forge maintainers: Is it possible to use `<` and `>` for restricting the version of `r_base`? The [docs](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build) show that it is possible to use `# [py<35]` for Python, so it would be convenient to have a similar option for R. While `[r_base != "3.6.3"]` gets the job done, it's pretty brittle because it depends on the exact versions of R that conda-forge currently has pinned. Something like `[r_base < "4.0"]` would be more future-proof and more closely matches the intent of the R package's actual restrictions.

**Comment by jdblischak** [comment]
@apcamargo Ack. Sorry. I confused myself and gave you bad advice. It looks like the CI passed because it built nothing. Could please change the skip line to be:

```
  skip: true # [r_base == "3.6"]
```

In other words, the recipe is skipped only if R is 3.6.x. Please use 3.6 and not 3.6.3. From my experiments in https://github.com/conda-forge/r-xml-feedstock/pull/19, the CI files for R 3.6 are only suppressed when specifying 3.6. I suspect the reason for the difference is because [conda-forge pins R to 3.6](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/67264d68770c28a6e144d4eca3939b21336979b0/recipe/conda_build_config.yaml#L625), but that is just a guess.

File: recipes/r-phylolm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    - r-future.apply
+
+test:
+  commands:
+    - $R -e "library('phylolm')"           # [not win]
+    - "\"%R%\" -e \"library('phylolm')\""  # [win]
+
+about:
+  home: https://github.com/lamho86/phylolm
+  license: GPL-2
```
Comment (dbast) [inline recipes/r-phylolm/meta.yaml:49]:
```suggestion
  license: GPL-2.0-or-later
```

spdx short for `# License: GPL (>= 2)` or cran meta data

File: recipes/r-phylolm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/phylolm_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/phylolm/phylolm_{{ version }}.tar.gz
+  sha256: d6fa45f58689cce9fe91519ab55a61c1e549e5a7f18b6989bf86234d483cff12
+
+build:
+  merge_build_host: True  # [win]
+  skip: True  # [r_base != "3.6.3"]
```
Comment (dbast) [inline recipes/r-phylolm/meta.yaml:17]: This skips the R4.0 build. Change to:

```suggestion
  skip: True  # [r_base != "3.6.3"]
```

**Comment by apcamargo** [comment]
Thanks ,@dbast

**Review by jdblischak — APPROVED** [review]

**Comment by dbast** [comment]
:)

**Pull request merged**